tests/expectations/tag/associated_in_body.c

```diff
@@ -3,9 +3,9 @@
 #include <stdint.h>
 #include <stdlib.h>
 
-/**
- * Constants shared by multiple CSS Box Alignment properties
```

Comment:
Is there a good reason to change the default behavior?

Reply:
I proposed it because it seemed slightly more in line with the most 'unopinionated' defaults I found online (c. [Wikipedia](https://en.wikipedia.org/wiki/Comment_(computer_programming)#Styles), [Wikibooks](https://en.wikibooks.org/wiki/C++_Programming/Code/Style_Conventions/Comments), and the [C++ reference](https://en.cppreference.com/w/cpp/comment)), and more tooling friendly than the "doxygen" style.

Whether that was a _good_ reason I don't know. As mentioned, I don't mind either way, just let me know what you want.

Reply:
As a general rule I'd prefer to keep the behavior the same and allow opting into the behavior change...

Though in this case the risk of breaking something unintendedly seems close to zero...  :)

Maybe @eqrion has opinions?

Reply:
I changed to default for `C`, added a README entry, and renamed `Doxylight` to `Doxy`.